### DISCUSSION TITLE
2023년 03월 26일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#21] [General]
# 2023년 03월 26일

[ 2023-03-01 ] [ 2023-03-02 ] [ 2023-03-05 ] [ 2023-03-06 ] [ 2023-03-07 ] [ 2023-03-08 ] [ 2023-03-09 ] [ 2023-03-10 ] [ 2023-03-12 ] [ 20…

https://cyma-s.github.io/2023-03-26/

**Comment by drunkenhw** [comment]
내일의 배로는 배로가 아니기 때문에 미루는 것도 참 좋죠~
저는 매번 미루는데 미래의 내가 잘 해결하더라고요~
ABN? 아시죠?

**Reply by Cyma-s** [reply]
박스터에게 선플 받으니 기분이 정말 좋네요 ~
ABN 명심하겠습니다~? 감사합니다~? ^ㅅ^

**Comment by ezzanzzan** [comment]
저도 베로를 보기 위해 주말 잠실.. 노력해보겠습니다.
성장해나가는 베로 아주 기특하고 귀엽고 멋있어요 최고 👍

**Reply by Cyma-s** [reply]
토리도 나오는 주말 잠실은 귀하네요 ^ㅅ^~
